chapters/bizleg-case-studies/startups/yellr.md

```diff
@@ -0,0 +1,208 @@
+
+## Sales objectives
+
+How can we increase our consumer demand?
+The problem of news organizations failing to retain and monetize upon their readers/viewers is not a new one and as time goes on, the need for a solution only increases. Yellr can allow news organizations to have a personal and customizable communication with their viewers while simultaneously increasing a single journalist's ability to collect sources and information from communities.
+
+How do we make sure the news organizations will want it, and that locals will use it-- increase engagement and download #s
+
+We have been in discussion with WXXI as well as many other news organizations such as WDKX which have given positive feedback on the product and have expressed interest in it. Our partnerships and collaboration will be imperative to the initial market penetration of yellr.  
+ 
+
+## Market Research Objectives
+
+Since the app is already available on the play store and the app store, we can simply run focus groups to better improve our app. Focus groups are better that simply asking an assignment because you are able to get a person's input in real time and user are able to explain their problems without having a character limit. 
+
+We have two main focus groups. News Organizations; they may be in a variety of forms, and also citizens that have an interest in local news. Since our product is a mobile app it has to have a “familiar” feel; familiar in the sense it is similar to other social media apps. “Generation X” is know to know their way around apps so it doesn’t have to be very easy to use.
+
+For internet marketing techniques we plan to use other social media outputs to create promotions. Facebook and Twitter can be used to update people on new assignments or even let them know when a Yellr event is happening near them.
+
+## Technological Environment 
+
+What will keep us ahead: Constant app updates, whether they are server updates, cosmetic updates, new features, etc. We will keep the question “What do you think of Yellr?” as a way to continuously gain feedback to improve our app.
+
+Security: Making sure we have preventative software as well as hiring security firms to find backdoors into our systems will be essential in protecting customer credit card numbers and location
+
+## R&D Objectives
+
+How will our product stay up to date?
+
+Since Yellr is open source, we will continually build in features as asked by the community and leverage an volunteer community of contributors. Customers of the Yellr Company can contract the Yellr Company to build features for them.
+
+Having an active volunteer contributor community will allow us to loss-lead out competitors in terms of feature development at a much smaller price. The use of the AGPLv3 and other copyleft software licenses protects our assets from being pulled into proprietary software projects and therefore covers any issues of people using our code as a derivative work.
```

Comment:
The point of FOSS isn't to produce *cheaper* software, it's to produce *better* software that meets user needs and puts control in the hands of a community.